web/js/panel.js

```diff
@@ -154,6 +154,8 @@ var requestPanelXML = function(panelName) {
         success: function(data, textStatus, jqXHR) {
             processPanelXML(data, textStatus, jqXHR);
             setTitle($gPanel["name"]);  // set final title once load completes, helps with testing
+            // set new attribute data-panel-name on the panel-area div to the panel name so that a user's css can use it.
+	        $("#panel-area").attr("data-panel-name", $gPanel["name"]);
```

Comment:
Minor issue: You have a TAB character at the beginning of the line, making the indentation wrong for other TAB sizes.

Reply:
Thanks, fixed.